assets/src/dashboard/utils/validateAdManagerSlotIdFormat.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// /123456789/a4a/amp_story_dfp_example
+const adManagerSlotIdFormatRegex = /^\/\d+(\/[\w_-]+)*$/;
+
+export default function validateAdManagerSlotIdFormat(value = '') {
+  return Boolean(value.toLowerCase().match(adManagerSlotIdFormatRegex));
```

Comment:
It'd probably be good to eventually add some unit tests for these regex

Reply:
Added some tests now 👍